doc/src/tutorial/compile_cmake.dox

```diff
@@ -1,57 +1,251 @@
+
+@subsection compile_cmake_general_running_tests_examples Running Tests and Examples
+
+As stated earlier, tests and examples are located within the `test` and `examples` folders, respectively. To list tests, run the following command in the same terminal or command prompt window used earlier (substitute `dir` at the Windows command prompt):
+
+    ls test/
+
+Alternatively, substitute `test/` with `examples/` for examples instead. To actually run a test, run the following command:
+
+    test/patest_sometest
+
+Replace `patest_sometest` with the name of the test you want to run.
+
+The same applies to examples:
+
+    ls examples/ # List all known examples
+    examples/paex_sine # Execute an example
+
+@subsection compile_cmake_general_options Build Options
+
+PortAudio's CMake build system also features several configuration options. For a list of options, see @ref compile_cmake_options. Append them to your configure command in the format `-DOPTION_NAME=VALUE`, with `VALUE` usually being either `ON` or `OFF`. For example, `cmake /path/to/portaudio -DPA_BUILD_TESTS=ON`. Note that you should not need to restart with a fresh, empty build directory each time you change a config option, but if you encounter problems, then you should start with a fresh build directory and try again.
+
+If you want to use a different build file generator, add `-G "Generator Name"` to the initial configuration `cmake` command, where "Generator Name"` names the generator that you want to use, for example "Ninja". We will only cover the default generators (Visual Studio projects and Unix Makefiles) here, but if you are experienced with other generators, you are free to use them. The command `cmake --help` lists all available build file generators.
+
+The following sections give platform-specific examples. Note that these examples assume that you have created a build directory outside the source tree named `pabuild`.
+
+@section compile_cmake_win Windows
+
+On Windows, CMake will generate Visual Studio projects by default, so you will need to install Visual Studio first. When installing Visual Studio, ensure you have the **Desktop Development with C++** workload selected. Then, for a 64-bit build, open a command prompt (it does not need to be a developer command prompt in this case). Alternatively, you may also use PowerShell instead if you choose as the commands below will also work in PowerShell.
+
+To get started, in your command prompt or PowerShell window, run the following commands:
+
+    C:\> mkdir pabuild
+    C:\> cd pabuild
+
+Then, run the following command to configure the build:
+
+    C:\pabuild> cmake C:\path\to\portaudio
+
+Once configuration is successful, you can then start the build by running the following command:
+
+    C:\pabuild> cmake --build .
+
+This builds a _debug_ config of PortAudio. If you need a release config of PortAudio, add `--config Release` to the above command. For more CMake options, see @ref compile_cmake_options_general.
```

Comment:
```suggestion
This builds the Debug configuration within the Visual Studio solution. If you need a release build of PortAudio, add `--config Release` to the above command. For more CMake options, see @ref compile_cmake_options_general.
```

Reply:
`--config` is correct. I can't recall where I got `--configuration`, so that'll be fixed. Everything else mentioned here will be incorporated also! :smile: